### ISSUE TITLE
Chapter6 - The maximum value of the register

### ISSUE THREAD

**Issue by GGGGGHT** [lizrice/learning-ebpf#48]
> Register 2 doesn’t have a minimum value, and the umax_value shown here in decimal corresponds to 0xFFFFFFFF, which is the largest value that can be held in an 8-byte register. In other words, at this point the register could hold any of its possible values.
In instruction 4, the contents of Register 2 are copied into Register 3, and then instruction 5 adds one to that value. Therefore, Register 3 could have any value that’s 1 or greater. You can see this in the state information for Register 3, which has umin_value set to 1, and a umax_value of 0xFFFFFFFF.

The maximum value that 8 bytes can represent should be `0xFFFFFFFFFFFFFFFF`, not `0xFFFFFFFF`. I'm not sure if my understanding is wrong.

**Comment by lizrice** [comment]
You're right, that is an error! Thanks for catching, I'll need to fix that as an errata

**Comment by lizrice** [comment]
I'll leave this open to remind me to fix it next time I'm in the O'Reilly errata portal :-)
